src/utils.js

```diff
@@ -76,6 +76,10 @@ export function getSchemaType(schema) {
     return "string";
   }
 
+  if ((!type && schema.properties) || schema.additionalProperties) {
```

Comment:
Is this necessary? Could we just make it

```suggestion
  if (schema.properties || schema.additionalProperties) {
```
Or does it need to be
```suggestion
  if (!type && (schema.properties || schema.additionalProperties)) {
```

Reply:
Agree the latter is better. Making that change!